tools/helium.py

```diff
@@ -1487,21 +1488,21 @@ def parse_case_statement(self):
                 self.i += 1
             self.expect(DO)
             statements = []
-            while self.tokens[self.i] is not WHEN and self.tokens[self.i] is not ELSE and self.tokens[self.i] is not END and self.tokens[self.i] is not END_OF_FILE:
+            while self.tokens[self.i] is not WHEN and self.tokens[self.i] is not OTHERS and self.tokens[self.i] is not END and self.tokens[self.i] is not END_OF_FILE:
```

Comment:
You've changed this differently from how you changed the C++ version. They're both wrong. Make this one match the fixed C++ one.